kvstore/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "solana-kvstore"
+description = "Embedded Key-Value store for solana"
+version = "0.0.1"
+homepage = "https://solana.com/"
+readme = "../README.md"
```

Comment:
IMO remove this readme entry, we only need this in one crate (core)

Reply:
done